code/datums/components/deadchat_control.dm

```diff
@@ -191,9 +194,9 @@
 	examine_list += span_notice("[A.p_theyre(TRUE)] currently under deadchat control using the [(deadchat_mode & DEMOCRACY_MODE) ? "democracy" : "anarchy"] ruleset!")
 
 	if(deadchat_mode & DEMOCRACY_MODE)
-		examine_list += span_notice("Type a command into chat to vote on an action. This happens once every [input_cooldown * 0.1] seconds.")
+		examine_list += span_notice("Type a command into chat to vote on an action. This happens once every [CEILING(input_cooldown * 0.1, 1)] second\s.")
 	else if(deadchat_mode & ANARCHY_MODE)
-		examine_list += span_notice("Type a command into chat to perform. You may do this once every [input_cooldown * 0.1] seconds.")
+		examine_list += span_notice("Type a command into chat to perform. You may do this once every [CEILING(input_cooldown * 0.1, 1)] second\s.")
```

Comment:
Why all these CEILINGS? a cooldown of 0.1 seconds would be fine here

Reply:
ok, i got rid of them on everything except the message when you try to submit a command too early